### ISSUE TITLE
Tunes Archive

### ISSUE THREAD

**Issue by edwardabraham** [slow-session/wellingtonsession.org#19]
On reflection I am not quite sure about the "Tunes Archive" page at the moment. I feel like it is only really useful if it is comprehensive, and that depends on whether we can achieve that before launch. And of course comprehensive is pretty much impossible!

I also strongly feel that a deluge of hundreds of tunes is of limited use (and can have a negative effect on people by overwhelming them). If you are able to learn hundreds of tunes, then you probably don't need this site. The real benefit of the slow session site, is, I think, in the curation. 

It may also be good to limit the amount that we need to maintain at the moment.

So, I guess I am proposing not having the Tune Archive page before launch.

**Comment by tcizadlo** [comment]
I disagree :)  I think I'd like a place for putting new tunes as we find them, not clogging up the list.  I also feel this can be a resource for 'fast' session players to learn new tunes also.  I think we should keep it, even if it is just a long list at first. (in case the search box proves difficult).

**Comment by edwardabraham** [comment]
I guess I would rather have a shorter, well-curated list that is relevant,
rather than a long unfocussed list. Especially for an initial launch.

Of course, you may be able to make a long, well-curated list, but be
careful not to bite too much off all at once!

**Comment by edwardabraham** [comment]
Another approach is to have all the tunes on one page, but to use the search function to filter for current tunes, and all tunes. Then we only have one page which is a big list of tunes, which I  think I would prefer.

**Comment by tcizadlo** [comment]
I completely agree - I think the single page of tunes would be the best, although I'd suggest we default to "current tunes", in the search filter (perhaps allow new (10 or so), current, and all?).  I do think the front page should be a near variant of what we have, with the header shrunk and reorganised.
